go/tasks/plugins/presto/client/presto_client.go

```diff
@@ -0,0 +1,66 @@
+package client
```

Comment:
This is just a noop Presto client for the open-source version. There will be a separate commit for one based on Mozart